en/docs/design-your-project/index.md

```diff
@@ -1,4 +1,4 @@
-# Architecture Diagram
+# Design your project
```

Comment:
Image in `Access the Architecture View` is too big.

Reply:
Fixed.